.github/workflows/build-deploy.yml

```diff
@@ -26,22 +26,37 @@ jobs:
   #         version: v1.60.3
   #         args: --timeout 600s --verbose ./...
 
-  test:
+  codacy-analysis:
+    if: github.repository == github.event.pull_request.head.repo.full_name
```

Comment:
Can you move this condition to just the codacy steps (steps have ifs too)? It still makes sense to run tests for forks imo

Reply:
Done